os/arch/arm/src/artik053/Kconfig

```diff
@@ -110,6 +110,30 @@ config ARTIK053_AUTOMOUNT_USERFS_MOUNTPOINT
 		Specifies the mount point where user r/w file system
 		will be mounted at.
 
+config ARTIK053_AUTOMOUNT_ROMFS
```

Comment:
Automount functionality is a different kind of changes. Let's split commits.

Reply:
done